### ISSUE TITLE
Two bugs causing failure on accessibility

### ISSUE THREAD

**Issue by ohassanx** [Splidejs/vue-splide#101]
### Checks

- [X] Not a duplicate.
- [X] Not a question, feature request, or anything other than a bug report directly related to Vue Splide. Use Discussions for these topics: https://github.com/Splidejs/splide/discussions

### Version

0.6.12

### Description

These are the two errors that is being shown when running lighthouse on a page that currently uses vue-splide.
![Screenshot 2023-11-15 at 08 51 02](https://github.com/Splidejs/vue-splide/assets/45073967/5f828d8f-7f17-4662-91e5-128aae14cd5a)
This specific error comes from having the li elements have the role="group" whereas the parent ul element has the role="presentation" which is not compatible as it does not pass the aria-role validation. https://dequeuniversity.com/rules/axe/4.8/aria-roles
![Screenshot 2023-11-15 at 08 51 11](https://github.com/Splidejs/vue-splide/assets/45073967/108cd9fd-10dd-4cde-8252-240ddc86126d)
This specific error comes from having aria-hidden="true" on slides that are currently not in view.

### Steps to Reproduce

1. Run lighthouse on a page that uses vue-splide.

### Expected Behaviour

To not have these two errors in lighthouse. 
To remove aria-hidden="true" from slides that are not in view.
To have the correct role for the ul element and not presentation so the correct role in this case would be role="Widget".
